browser/src/VariantList/filterVariants.ts

```diff
@@ -91,8 +91,10 @@ const filterVariants = (variants: Variant[], filter: VariantFilterState, selecte
 
   filteredVariants = filteredVariants.filter((v: Variant) => v.exome || v.genome)
 
-  if (filter.searchText && !filter.includeContext) {
-    filteredVariants = getFilteredVariants(filter, variants, selectedColumns)
+  if (filter.searchText) {
```

Comment:
Here I think we may need one more branch, to handle the unchecking of "display neighboring variants".

With this change, this re-enables the behavior of allowing the user to decide whether or not to include the filtered variants (via the "Filtered variants" checkbox), but disables the behavior of allowing the user to filter the list to have just the rows that are a partial match (via the "Display neighboring variants" checkbox).

E.g., in the screenshot below, the "display neighboring variants" checkbox is unchecked, so we'd expect the variant list to only include rows that match the search terms

![Screenshot 2024-09-17 at 10 24 57](https://github.com/user-attachments/assets/8d94871f-fd49-4a62-ad34-45166fc89dc0).